Prog/TOPA/TOPA_CBO_spreadsheet.sas

```diff
@@ -35,19 +35,23 @@ data Topa_CBO_sheet;
   add_notes=""; label add_notes = "Additional notes"; 
 run; 
 
-%File_info( data=Topa_CBO_sheet, printobs=5 ) 
+proc sort data=Topa_CBO_sheet out=Topa_CBO_sheet_sorted;
+  by u_address_id_ref descending u_notice_date;
```

Comment:
Not descending this time. Just normal order.

Reply:
@ptatian Yep - I'll change it back i just did that to look at the dedup_notices that way.